templates/validator/validatornotfound.html

```diff
@@ -22,7 +22,6 @@ <h1 class="h4 mb-1 mb-md-0">Validator not found</h1>
       <div class="card">
         <div class="card-body">
           <div class="d-1">Sorry, but we could not find the validator you are looking for.</div>
-          <div class="d-1">Stay calm, depending on the gas price of the transaction, <a href="https://kb.beaconcha.in/ethereum-2.0-and-depositing-process">it may still be in the mempool.</a></div>
```

Comment:
This error belongs here ... check the link - it explains the process of adding a validator

Reply:
yes ok, it makes sense now. I will add this message back, but I believe it could be phrased better to reflect that the initial validator's transaction has not yet reached the deposit contract. Will look at the message tho later.